pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -25,3 +25,14 @@ var TableDescriptorPollInterval = settings.RegisterDurationSetting(
 	1*time.Second,
 	settings.NonNegativeDuration,
 )
+
+// DefaultFlushFrequency is the default frequency to flush sink.
+// See comment in newChangeAggregatorProcessor for explanation on the value.
+var DefaultFlushFrequency = 5 * time.Second
+
+// TestingSetDefaultFlushFrequency changes defaultFlushFrequency for tests.
+// Returns function to restore flush frequency to its original value.
+func TestingSetDefaultFlushFrequency(f time.Duration) func() {
+	DefaultFlushFrequency = f
+	return func() { DefaultFlushFrequency = 5 * time.Second }
```

Comment:
super nit: missed this lat time, but I have a slight preference to saving the old value outside the closure and restoring it to inside the closure. Again, probably not worth another round of CI.